src/OpenTK/OpenTK.csproj

```diff
@@ -795,6 +806,7 @@
     <Compile Include="Platform\Linux\DefaultCursor.cs" />
     <Compile Include="Platform\Linux\Bindings\Kms.cs" />
     <None Include="OpenTK.csproj.paket.template" />
+    <None Include="packages.config" />
```

Comment:
This shouldn't be included, because we're using Paket to manage dependencies.

If a new dependency is needed for some reason, please use the Paket system.